dev/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:17.10
+
+ENV GOPATH /go
+ENV PATH ${GOPATH}/bin:/usr/lib/go-1.9/bin:$PATH
+
+# install golang 1.9
+RUN apt-get update && \
+    apt-get install -y software-properties-common && \
+    add-apt-repository ppa:gophers/archive && \
+    apt-get update && \
+    apt-get install -y curl golang-1.9-go autoconf git libio-captureoutput-perl python python-pip protobuf-compiler && \
+    rm -rf /var/lib/apt/lists/*
+
+# install ninka
+RUN mkdir /ninka && \
+    git clone https://github.com/dmgerman/ninka.git /ninka && \
+    cd /ninka/comments && make && make install && \
+    rm /usr/local/man/man1 && \
+    cd /ninka && perl Makefile.PL && make && make install && \
+    rm -fr /ninka
+
+# install scancode
+RUN pip install scancode-toolkit
+
+# install dgraph
+RUN curl https://get.dgraph.io -sSf | bash
+
+# install go deps
+RUN go get -u github.com/golang/protobuf/protoc-gen-go && \
+    go get github.com/dgraph-io/dgraph/client
```

Comment:
What about pflags? I really don't like managing dependencies this way.

Reply:
We should vendor our dependencies. Go dep it's the way to go. In this one i know that some other packages install pflags so it can be consider magic and opaque. Not very good indeed.